RedHatInsights/vulnerability-ui
# feat(152): adopts new pdf-generator

Enables using new PDF-generator. Pdf report on systems page has been removed, info here.

To test:

1. Run PR locally
2. navigate to cves page
3. click on export pdf from cves table toolbar
4. observe that pdf is downloaded and does not have major problem
5. Navigate to Reports page
6. try to generate different reports
